packages/now-build-utils/src/fs/run-user-scripts.ts

```diff
@@ -287,6 +287,11 @@ export async function runBundleInstall(
   assert(path.isAbsolute(destPath));
   const opts = { cwd: destPath, ...spawnOpts };
 
+  await spawnAsync(
+    'bundle',
+    ['config', 'path', 'vendor/bundle']
+  );
+
```

Comment:
This breaks jekyll and any other ruby static builds because `jekyll` is no longer in the `$PATH`.

```
bash: jekyll: command not found
```

Before this can be merged, we need to figure out how the relative path here can be appended to the system `PATH` so the subsequent call to `jekyll build` will find the installed jekyll binary.

Reply:
Another issue with installing to `./vendor/bundle`:

Invoking  `./vendor/bundle/ruby/2.7.0/bin/jekyll build` results in the following error:

```
Traceback (most recent call last):
	2: from ./vendor/bundle/ruby/2.7.0/bin/jekyll:23:in `<main>'
	1: from /ruby27/lib/ruby/2.7.0/rubygems.rb:294:in `activate_bin_path'
/ruby27/lib/ruby/2.7.0/rubygems.rb:275:in `find_spec_for_exe': can't find gem jekyll (>= 0.a) with executable jekyll (Gem::GemNotFoundException)
```